config/webpack.dev.config.js

```diff
@@ -1,4 +1,5 @@
 /* eslint import/no-extraneous-dependencies: 0 */
+const Dotenv = require('dotenv-webpack');
```

Comment:
How has the code been working so far without this? I'm assuming we have been environment variables before e.g. SENTRY.

Reply:
> How has the code been working so far without this? I'm assuming we have been environment variables before e.g. SENTRY.


I used to export environment variables via the terminal for testing purposes, and it always worked like for SENTRY. However, today, the same process didn't read the environment variables even after exporting them. After investigating further, we discovered that Webpack is not configured to read from the .env file.

Reply:
I think we're mixing up issues @VinneyJ .

1. Exporting env vars
2. Reading env vars from a file

You only need `2` if you aren't or can't do `1`. In prod, `1` is preferable e.g. in Dokku, we set the env vars on the Dokku app rather than provide an env file.